nominees/formpack.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Formpack",
+  "description": "Formpack is a tool used by kpi to parse and repackage survey data for exports, reports, and maps based on different languages available from the form definition.",
+  "website": "https://www.kobotoolbox.org/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/kobotoolbox/formpack/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 17,
+      "evidenceURL": "https://www.kobotoolbox.org/"
```

Comment:
@rachita19082 where possible for the `evidenceURL` field, try to link to blog or news articles as supporting evidence of the SDG in question. For example here is an [article](https://blogs.cisco.com/csr/a-digital-data-collection-tool-for-humanitarians) by Cisco that provides supporting evidence for SDG 17. Kindly update accordingly. Thanks.

Reply:
Resolved in commit ed736d9

Reply:
@rachita19082 you may want to pick out a specific article that supports the given SDG rather than provide a generic [link](https://www.kobotoolbox.org/blog/) like above. Thanks.